sbom.json

```diff
@@ -90,7 +90,7 @@
     ]
   },
   "serialNumber": "urn:uuid:acb30d08-ee47-4ff0-b301-d66ef1f54082",
-  "version": 1,
+  "version": 2,
```

Comment:
What is this version number?

Reply:
The version number in an SBOM is increased every time the SBOM changes. It doesn't hold a lot of relevance for us, but it's there.